charts/nx-cloud/values.yaml

```diff
@@ -1,257 +1,261 @@
-    registry: ''
-    imageName: nx-cloud-messagequeue
-    repository: ''
-    tag: ''
-    digest: ''
-    pullPolicy: Always
   deployment:
-    port: 61616
-    env: []
-    volumes: {}
-    volumeMounts: {}
-  service:
-    name: nx-cloud-messagequeue
-    type: ClusterIP
-    port: 61616
     annotations: {}
-  resources:
-    limits: {}
-    requests: {}
-
-nxCloudWorkflows:
-  enabled: false
-  port: 9000
-  name: 'nx-cloud-workflow-controller-service'
-  workflowsNamespace: 'nx-cloud-workflows'
-
-  # If externalName is left unset, the applications will look for ane existing service with the name defined
-  # by `nxCloudWorkflows.name` in the namespace `nxCloudWorkflows.workflowsNamespace`. Use this option if you are
-  # also running the nx-agents chart in the same cluster as this nx-cloud chart
-  #
-  # If externalName is set, an additional service will be created with the name `nxCloudWorkflows.name`
-  # in the global namespace of this chart, and applications will use that service to connect to the workflow controller.
-  # Use this option if your nx-agents are running in a different cluster than this nx-cloud chart
-  externalName: ''
-
-  # If you find that an externalName service is not working as expected, you can set this to true to create a headless service
-  # which will create an endpoint group as an alternative. Please continue to set `externalName` to the IP address
-  # you wish to direct traffic to as we will use it to populate the endpoint slice.
-  headless: false
-
-# Deprecated, use nxApi.deployment.replicas and frontend.deployment.replicas
-replicas:
-  frontend: 1
-  nxApi: 1
+    labels: {}
+    replicas: 1
+    port: 4203
+    env: []
 
-ingress:
-  skip: false
-
-  # deprecated, use ingress.annotations
-  globalStaticIpName: ''
-  managedCertificates: ''
-  albScheme: ''
-  albListenPorts: ''
-  albCertificateArn: ''
-  class: ''
-
-  annotations:
-    nx.app/installed-by: 'helm'
-
-fileStorage:
-  storageClassName: ''
-  size: '30Gi'
-  resourcePolicy: ''
-
-awsS3:
-  enabled: false
-  bucket: ''
-  accelerated: false
-  endpoint: ''
-  enablePathStyleAccess: false
-
-azure:
-  enabled: false
-  container: ''
-
-useCosmosDb: false
-
-gitlab:
-  apiUrl: ''
-  projectId: ''
-  auth:
-    enabled: false
-  mr:
-    enabled: false
+    strategy:
+      type: Recreate
+      # Uncomment below for RollingUpdate
+      # type: RollingUpdate
+      # rollingUpdate:
+      #   maxUnavailable: 0
+      #   maxSurge: 1
 
-github:
-  auth:
-    enabled: false
-  pr:
-    defaultWorkspaceId: ''
-    enabled: false
-    mode: 'webhook'
-    apiUrl: ''
+    envFrom: []
+    # envFrom:
+    # - configMapRef:
+    #     name: my-config-map
+    # - secretRef:
+    #     name: my-secret
 
-bitbucket:
-  auth:
-    enabled: false
+    # If you wish to steer placement of the api pod(s) these will be applied to the pod spec.
+    affinity: {}
+    tolerations: {}
+    nodeSelector: {}
 
-saml:
-  auth:
+    resources: {}
```

Comment:
Just include defaults if we have an idea of a baseline

Reply:
added the defaults from the previous version of the chart